config/routes.rb

```diff
@@ -274,4 +288,5 @@
   match ':controller(/:action(/:id))(.:format)'
```

Comment:
Yes, we could set an issue asking people to pitch in on those remaining 20 routes, but for the time being it is the lowest-priority route, so we should be fine. Thanks.